### ISSUE TITLE
Flaky test

### ISSUE THREAD

**Issue by monperrus** [gnieh/bluelatex#241]
Test "Scenario: retrieving the last modification date for modified paper should return new date" is flaky.

Sometimes it passes, sometimes it doesn't

**Comment by satabin** [comment]
Yes, this is the problem with such kind of tests… I do not see any reasonable way to do this and would really like to avoid pausing the tests during 10 seconds just to be sure the time delta is big enough.
